content/en/docs/v3.6/learning/api.md

```diff
@@ -323,6 +323,10 @@ message ResponseOp {
 }
 ```
 
+For `response_range` and `response_delete_range` responses in `TxnResponse`, the `revision` included in the inner response header is just the revision when generating the response, so it might be one less than the `revision` included
```

Comment:
Actually all responses, without clarification. And also without clarification about "one less".

I would write something like:

```
Response headers returned in TXN should not be interpreted in any way, and should be completely ignored.
```

Reply:
Thanks for the feedback. Just updated the doc. 

It isn't decided to deprecate the inner `ResponseHeader` for now. Please note that each response has exactly the same format as non-TXN response. 

The `revision` included in each response might be still useful, such as checking whether the deletion is successful or not when there is no PUT request before it.

Reply:
Makes sense to me & updated & thx.

Note that there is no plan to remove the `ResponseHeader` in each inner response. See https://github.com/etcd-io/etcd/issues/14490#issuecomment-1257086651